qgis_deployment_toolbelt/profiles/rules_context.py

```diff
@@ -60,6 +68,29 @@ def _context_date(self) -> dict:
             "current_year": today.year,
         }
 
+    @property
+    def _context_env(self) -> dict:
+        """Returns a dictionary containing environment variables that can be used in
+            QDT various places: rules...
+
+        The environment variables can be filtered based on self.only_prefixed_variables
+        and self.variables_prefix settings.
+
+        Returns:
+            dict: dict with environment variables to use in rules.
+        """
+        env_vars = dict(os.environ)
```

Comment:
It's already a mapping object no?

https://docs.python.org/3/library/os.html#os.environ